### ISSUE TITLE
release versions are using marty 0.9.17 instead of 0.10.x

### ISSUE THREAD

**Issue by TN1ck** [jhollingworth/marty-express#11]
When trying to install the `0.4.0` of marty-express, npm gives this error:

```
npm ERR! argv "node" "/usr/local/bin/npm" "install" "marty"
npm ERR! node v0.10.35
npm ERR! npm  v2.1.18
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package marty does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer marty-express@0.4.0 wants marty@^0.9.17

```

**Comment by jhollingworth** [comment]
Have you got Marty 0.9.7 installed? It got released on 26th

**Comment by TN1ck** [comment]
No, 0.10.1. I fixed with cloning marty-express and doing `npm link` and `npm link marty-express`. I don't know why the `package.json` is different from the one in the repository. But when you click on the commit the `0.4.0` release is reffering to, you see marty `0.9.17`: https://github.com/martyjs/marty-express/blob/7f3d7a7d4925689060ab0e10ba8c6bcca53ce873/package.json

**Comment by jhollingworth** [comment]
can you try upgrading to marty-express v0.10?

**Comment by TN1ck** [comment]
Oh I was under the impression that `0.4` is the version that supports marty `0.10.x.`0.10` is working fine.

**Comment by jhollingworth** [comment]
sadly not, v0.4 was released for Marty v0.9. I've moved version numbers of other marty repos to be in line with marty to avoid this confusion going forwards

**Comment by TN1ck** [comment]
Ah ok, Sorry then! The rapid succession of the release-dates got be thinking they are both for 0.10. This issue can be closed.
